shared/src/packages/trucks/libs/types/truck-entity.type.ts

```diff
@@ -0,0 +1,12 @@
+type TruckEntity = {
+  id: number;
+  createdAt: Date;
+  updatedAt: Date;
```

Comment:
for backend when retrieved from DB, it will be Date, but though network it will be passed as string. So on frontend it will be string